.github/workflows/stale-bot.yml

```
@@ -0,0 +1,31 @@
+name: "Mark and close stale issues and PRs"
+on:
+  workflow_call:
+
+permissions:
+  issues: write
+  pull-requests: write
+
+env:
+  STALE_ISSUE_MSG: "This issue has been marked as Stale because it has been open for 180 days with no activity. If you would like the issue to remain open, please comment on the issue or else it will be closed in 30 days."
+  STALE_PR_MSG: "This PR has been marked as Stale because it has been open with for 180 days with no activity. If you would like the PR to remain open, please comment on the PR or else it will be closed in 30 days."
+  CLOSE_ISSUE_MSG: "Although we are closing this issue as stale, it can be reopened if there is renewed community interest. Just add a comment to notify the maintainers."
+  CLOSE_PR_MSG: "Although we are closing this PR as stale, it can still be reopened to continue development. Just add a comment to notify the maintainers."
+
+jobs:
+  stale-awaiting-response:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v9
+        with:
+          stale-issue-message: ${{ env.STALE_ISSUE_MSG }}
+          stale-pr-message: ${{ env.STALE_PR_MSG }}
+          close-issue-message: ${{ env.CLOSE_ISSUE_MSG }}
+          close-pr-message: ${{ env.CLOSE_PR_MSG }}
+          close-issue-reason: ${{ env.CLOSE_ISSUE_REASON }}
+          days-before-stale: 1 # 1 for testing but we would want it to be more like 180
+          days-before-close: 1 #using 1 for testing, but before merge we want it to be more like 30
```

I find 180 days is too long. I believe shortening the window to 90 days and giving another 30 is an appropriate window. Let me know if you disagree.

```suggestion
          days-before-stale: 90
          days-before-close: 30
```

---

No strong opinion here--agreed!